### PR TITLE
Reorganize DBM Oracle setup docs

### PR DESCRIPTION
**Pull request by jhgilbert**
### What does this PR do?
Reorganizes the DBM Oracle setup docs to make them easier to follow.

### Motivation
DOCS-5987

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/database_monitoring/setup_oracle/_index.md [hunk 1]
```diff
@@ -85,52 +90,66 @@ Set `APIKEY` and run the following command in the command prompt inside the dire
...
+{{< /tabs >}}
 
-The following beta builds contain implemented Oracle DBM features:
-- Linux: `7.46.0~dbm~oracle~beta~0.33-1`
-- Windows: `7.46.0-dbm-oracle-beta-0.33-1`
-- Docker: `7.46.0-dbm-oracle-beta-0.33`
+### Upgrade an existing Agent installation
 
-### Oracle client
+<div class="alert alert-info">For the latest Oracle monitoring features and bug fixes, <a href="/database_monitoring/setup_oracle/?tab=linux#recommended-agent-version">install the recommended beta version</a> instead of upgrading your existing Agent below.</div>
```
Comment (caseycull) [inline content/en/database_monitoring/setup_oracle/_index.md:112]: I feel that this section doesn't capture the scenario where someone may be upgrading from a custom build to a new custom build or from an official version to a custom build

**Review by caseycull — CHANGES_REQUESTED** [review]

File: content/en/database_monitoring/setup_oracle/_index.md [hunk 2]
```diff
@@ -19,49 +19,51 @@ The features described on this page are in beta. Contact your Customer Success M
...
 
-Since the Agent doesn't require an external Oracle client, remove the `jdbc_driver_path` configuration parameter from the new parameter file `/etc/datadog-agent/conf.d/oracle-dbm.d/conf.yaml`.
+## Prerequisite details
 
 ### Install the Agent
 
-To start collecting Oracle telemetry, first install the Datadog Agent.
+#### Host requirements
 
 The Agent does not need to run on the same server nor the same platform as the monitored databases, and can be installed remotely using one of the suggested [setup architectures][10].
```
[the remark this answers is not in the document]
Comment (nenadnoveljic) [reply content/en/database_monitoring/setup_oracle/_index.md:45]: > I think this part is misleading: `The Agent does not need to run on the same server nor the same platform as the monitored databases ...`
> 
> since in self-hosted setups, the Agent is in fact running on the same server

Not necessarily - the Agent still can be running on a different server. For example, it is possible to install a single agent monitoring different databases on different hosts. In fact, some customers explicitly asked for that.

File: content/en/database_monitoring/setup_oracle/_index.md [hunk 3]
```diff
@@ -19,49 +19,51 @@ The features described on this page are in beta. Contact your Customer Success M
...
 
-Datadog recommends you install the version listed in the [Latest Agent version](#latest-agent-version). It contains all the implemented Oracle monitoring features and bug fixes.
+#### Recommended Agent version
+
+Datadog recommends you install the beta version listed below, as these contain all the implemented Oracle monitoring features and bug fixes:
```
[the remark this answers is not in the document]
Comment (nenadnoveljic) [reply content/en/database_monitoring/setup_oracle/_index.md:49]: I also think we should remove `beta`.

File: content/en/database_monitoring/setup_oracle/_index.md [hunk 4]
```diff
@@ -19,49 +19,51 @@ The features described on this page are in beta. Contact your Customer Success M
...
+- Linux: `7.46.0~dbm~oracle~beta~0.33-1`
+- Windows: `7.46.0-dbm-oracle-beta-0.33-1`
+- Docker: `7.46.0-dbm-oracle-beta-0.33`
+
+If you prefer an official Datadog Agent release, a minimum version of `7.46.0` is recommended.
 
-If the latest Agent version is an official Datadog Agent release, like `7.46.0`, follow the instructions in [Official release](#official-release). If the latest Agent version is a beta build, such as `7.46.0~dbm~oracle~beta~0.33`, follow the instructions in [Beta build](#beta-build).
```
[the remark this answers is not in the document]
Comment (nenadnoveljic) [reply content/en/database_monitoring/setup_oracle/_index.md:56]: > except in the version tag itself)
I would remove it from there too, as it's just an example.

File: content/en/database_monitoring/setup_oracle/_index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nenadnoveljic) [reply content/en/database_monitoring/setup_oracle/_index.md:49]: But is it possible that this branch is created from an older `master`, as the references to `beta` have already been changed?

File: content/en/database_monitoring/setup_oracle/_index.md [hunk 5]
```diff
@@ -19,49 +19,51 @@ The features described on this page are in beta. Contact your Customer Success M
...
+- To install a beta build, see [Beta build installation](#beta-build-installation).
+- To install the latest official release, follow the [instructions for your platform][3]. 
 
-#### Official release
+#### Oracle client
```
[the remark this answers is not in the document]
Comment (nenadnoveljic) [reply content/en/database_monitoring/setup_oracle/_index.md:59]: Yes, that's better.

File: content/en/database_monitoring/setup_oracle/_index.md (hunk 1 above)
Comment (caseycull) [reply content/en/database_monitoring/setup_oracle/_index.md:112]: could you say more about why you added these parts?
image

Comment (caseycull) [reply content/en/database_monitoring/setup_oracle/_index.md:112]: could you say more about why you added these parts?
image

File: content/en/database_monitoring/setup_oracle/_index.md [hunk 6]
```diff
@@ -85,54 +87,68 @@ Download the MSI file for the [Oracle DBM build][4].
...
 ```
+[4]: https://s3.amazonaws.com/ddagent-windows-stable/beta/datadog-agent-7.47.0-dbm-oracle-0.40-1.x86_64.msi
+[8]: https://ddagent-windows-stable.s3.amazonaws.com/
 
-##### Docker
-
-The docker Oracle DBM images can be found [here][9].
+{{% /tab %}}
+{{% tab "Docker" %}}
+The Oracle DBM images for Docker can be found [here][9].
```
Comment (jhgilbert) [inline content/en/database_monitoring/setup_oracle/_index.md:97]: @caseycull I realized that we're not supposed to use "here" links, and this should probably be a clearer instruction anyway. When we link these build repositories, are we asking them to download the build they need, then run the shell command in the folder with the downloaded build? Same question for all of these instances that say something similar about a build repository.

File: content/en/database_monitoring/setup_oracle/_index.md [hunk 7]
```diff
@@ -19,66 +19,68 @@ The features described on this page are in beta. Contact your Customer Success M
...
+    - [Install the Agent](#install-the-agent)
+    - [Upgrade an existing Agent installation](#upgrade-an-existing-agent-installation)
+2. The Oracle integration must be installed.
+    - [Install the Oracle integration](#install-the-oracle-integration)
+    - [Verify an existing Oracle integration installation](#verify-your-existing-oracle-integration)
 
-Deactivating the legacy integration prevents sending the system metrics twice.
+### Setup
+If the above prerequisites are met, follow the setup instructions for your hosting type:
+{{< partial name="dbm/dbm-setup-oracle" >}}
```
Comment (apigirl) [inline content/en/database_monitoring/setup_oracle/_index.md:37]: This is probably scope creep for this PR and could be an iterative update, but it seems like we could get more specific images for this partial. It's repetitive to the point where there's not really a point to having it in a partial. 
image

Comment (jhgilbert) [reply content/en/database_monitoring/setup_oracle/_index.md:37]: I added a ticket to my queue, thank you!
